pilot/pkg/model/push_context_test.go

```diff
@@ -2799,6 +2799,113 @@ func TestInitVirtualService(t *testing.T) {
 	})
 }
 
+func TestVirtualServiceOrder(t *testing.T) {
```

Comment:
add this case in `TestMergeVirtualServices`

Reply:
done